cognite/tests/containers_tests.rs

```diff
@@ -0,0 +1,69 @@
+use std::collections::HashMap;
```

Comment:
Should this maybe be flagged as an integration test?
```suggestion
#![cfg(feature = "integration_tests")]

use std::collections::HashMap;
```